var/spack/repos/builtin/packages/mathematica/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Mathematica(Package):
+    """Mathematica: high-powered computation with thousands of Wolfram Language
+       functions, natural language input, real-world data, mobile support.
+
+       Note: A manual download is required for Mathematica.
+       Spack will search your current directory for the download file.
+       Alternatively, add this file to a mirror so that Spack can find it.
+       For instructions on how to set up a mirror, see
+       http://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "https://www.wolfram.com/mathematica/"
+    url = 'file:///{0}/Mathematica_12.0.0_LINUX.sh'.format(os.getcwd())
```

Comment:
Isn't this one too many slashes since `os.getcwd()` returns a directory with a slash at the beginning?

Reply:
Oops, that's right -- fixed